eng/pipelines/runtime-extra-platforms-wasm-only.yml

```diff
@@ -0,0 +1,225 @@
+jobs:
```

Comment:
We could share the same yml file and just condition with `${{ }}` based on definition name. 

Look at: https://github.com/dotnet/runtime/blob/59e643b9932122a12e71ceb360a9ae3bb9420dec/eng/pipelines/libraries/outerloop.yml#L24

Reply:
hm I'll have to leave that one to you

Reply:
Never mind, you are already doing this but with an extra layer (using template references).